Assets/Scripts/Main.asmdef

```diff
@@ -3,7 +3,8 @@
     "rootNamespace": "OperationBlackwell",
     "references": [
         "GUID:f039f22f2b49b14ccbd1ae39fcf00fdb",
-        "GUID:6055be8ebefd69e48b49212b09b47b2f"
+        "GUID:3993cb11d8b1b5dbc97da6b4d9c3cd81",
+        "GUID:6ba797a32bdc954729fd08c85af0fff8"
```

Comment:
I assume this is for the `Player.asmdef` and `UI.asmdef` that were added? If so, what was removed then?

Reply:
The additions are player and UI yes, the removal was the saving system if im correct. And yes i tested it, but be sure to check it again.

Reply:
It's TextMeshPRO and the change makes sense.